PushNotifications.md

```diff
@@ -302,6 +302,16 @@ On iOS, to show alert notifications, you need to request provisional permission
 
 On Android, you can use logcat built into Android Studio or [pidcat](https://github.com/JakeWharton/pidcat) to view the logs.
 
+### I have confirmed messages are not being received by the device (no logs of being cancelled), but no errors are thrown when I sent the message.
+
+Sending an Ably message with a push payload will succeed if the message is successfully delivered to Ably, including delivery guarantees for channel subscriptions. However, push notification messages may not be delivered to specific devices if there is an error returned by APNs/ FCM. You should retrieve the device registration using a Push Admin or the Ably Dashboard. For example, if your application has a bundle ID which does not match APNs certificate generated from developer.apple.com,  you would get `DeviceTokenNotForTopic`, on the Ably dashboard, this would look like:
```

Comment:
Spacing around '/', please. Also, perhaps worth making the Apple domain a link.

```suggestion
Sending an Ably message with a push payload will succeed if the message is successfully delivered to Ably, including delivery guarantees for channel subscriptions. However, push notification messages may not be delivered to specific devices if there is an error returned by APNs / FCM. You should retrieve the device registration using a Push Admin or the Ably Dashboard. For example, if your application has a bundle ID which does not match APNs certificate generated from [developer.apple.com](https://developer.apple.com/account/resources/identifiers/list), you would get `DeviceTokenNotForTopic`, on the Ably dashboard, this would look like:
```